### ISSUE TITLE
[`blocks-dev`] file input is broken

### ISSUE THREAD

**Issue by abidlabs** [gradio-app/gradio#900]
### Describe the bug

Tried running these two demos:
* `demo\zip_two_files\run.py`
* `demo\zip_to_json\run.py`

But they were unrunnable because the file input is broken:
* Before you select the file, it says "drop an audio file" instead of "drop a file"
* After you drop the file, the file takes over the whole screen making you unable to click submit (see screenshot)

### Reproduction

* `demo\zip_two_files\run.py`
* `demo\zip_to_json\run.py`

### Screenshot

<img width="873" alt="image" src="https://user-images.githubusercontent.com/1778297/160499470-3876b557-3426-4af0-9131-d9d450b24c80.png">

### System Info

```shell
blocks-dev
```

### Severity

blocker

**Comment by abidlabs** [comment]
The output file component does not work. Try running `demo\zip_two_files\run.py`, or the code below:

```python
import gradio as gr


def zip_two_files(file1, file2):
    return file1
    # with ZipFile("tmp.zip", "w") as zipObj:
    #     zipObj.write(file1.name, "file1")
    #     zipObj.write(file2.name, "file2")
    # return "tmp.zip"


demo = gr.Interface(
    zip_two_files,
    ["file", "file"],
    "file",
    examples=[
        ["files/titanic.csv", "files/titanic.csv"],
    ],
)

if __name__ == "__main__":
    demo.launch()

```

**Comment by pngwn** [comment]
Closed by #931.
